.github/workflows/check.yml

```diff
@@ -1,53 +1,82 @@
-name: check
+name: Check
 
-on: [pull_request, push]
+on:
+  push:
+  pull_request:
+    branches:
+      - master
 
 jobs:
+  check_duplicate_runs:
```

Comment:
Now this is interesting, I wasn't aware of this action. Yes I saw the builds are duplicated sometimes but at this scale I guess it never became an issue. Seems to be a great idea with just one question - this config serves not just as a working ex_check CI config but also as an example for others using ex_check on CI and as such it's linked in README. I wonder if we may assume that this pattern and the way you've implemented it here is a good reference and default for most Elixir projects. What do you think?

Reply:
Well, keeping this step of CI is actually up to you. IMHO, this Github Actions behavior about running the same actions twice is something that people should know how to avoid.

Reply:
Ok let's keep it more for reference of others than for the actual need of this repo.